extensions/tika/runtime/pom.xml

```diff
@@ -43,6 +43,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
@galderz Thanks for this PR. My only reservation is that with `quarkus-tika`, if a user configures a PDF parser only for example (`quarkus.tika.parser=pdf`)), then so far none of the non-PDF dependencies are expected to make it to the native image. 
I wonder, if this dependency were optional, then would making a build step which would check if `OOXML` is enabled (can be true by default to start with - I can add an `ooxml` shortcut later) and if yes - add a native resource build item which would make this `ooxml-schemas` included ?
Is it realistic ?

Reply:
This dependency is only needed during building of the image itself. It's only for class resolution while trying to discover which methods the class has, nothing else. None of the classes that were missing would actually make it to the final image.

Reply:
IOW, it has 0 impact on the pointsto analysis that svm does. So, unless the end application uses something there indirectly, nothing will be included in the native image.

Reply:
Hi @galderz, so for example, `org/openxmlformats/schemas/wordprocessingml/x2006/main/STDisplacedByCustomXml` is missing, and few more I guess, so adding  this dependency also contributes to resolving #7198 (see for example https://github.com/quarkusio/quarkus/pull/7198/files#diff-cefda81cecd3360b76139d0044ecb53aR25). So unless `quarkus-tika` processor references these classes directly or indirectly, no native image size will be affected ?

Reply:
@galderz cool, your last comment makes it quite clear, cheers